docs/app-starter.stories.mdx

```diff
@@ -53,17 +53,19 @@ The order of these CSS stylesheets is **very important**:
 
 No matter how you choose to import the following, this order must be upheld.
 
+**We recommend you use your `tailwind.css` file to ensure this order (JS files will order them differently)**
```

Comment:
Below this same document says:
> You can either import this via JS or CSS,

Do we no longer recommend that? If so, recommend updating that other wording.